### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by RahullRawat**
-Add to archive
Restore from archive
Delete from archive

https://deploy-preview-7--topnote.netlify.app/

### CODE REVIEW

File: src/Services/addToArchive.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		const response = await axios.post(
+			`/api/notes/archives/${note._id}`,
+			{ note },
+			{ headers: { authorization: token } }
+		);
+		if (response.status === 201) {
+			notesDispatch({ type: "ARCHIVE_NOTE", payload: response.data });
+		} else {
+			throw new Error();
+		}
```
Comment (Abhijeet43) [inline src/Services/addToArchive.js]: You can also specify an error message to make it more understandable to user...What kind of error it is

**Review by Abhijeet43 — CHANGES_REQUESTED** [review]
deploy-preview-7--topnote netlify app_home

On adding new note...Color Pallete gets open automatically

**Review by kart-c — APPROVED** [review]
- Code is simple and easy to understand.
- Just a suggestion, the indenting looks off, you can use prettier extension to fix that.

**Comment by RahullRawat** [comment]
> * Code is simple and easy to understand.
> * Just a suggestion, the indenting looks off, you can use prettier extension to fix that.

But I am using prettier 😥

**Review by Abhijeet43 — APPROVED** [review]
- The functionality is working as intended!!!
- Code is clean
- Suggestion: Formatting code can help...Try indenting well

**Pull request merged**